cassandra/policies.py

```diff
@@ -572,8 +573,9 @@ def __init__(self, hosts):
         self._allowed_hosts = tuple(hosts)
         self._allowed_hosts_resolved = []
         for h in self._allowed_hosts:
-            if isinstance(h, UnixSocketEndPoint):
-                self._allowed_hosts_resolved.append(h._unix_socket_path)
+            unix_socket_path = getattr(h, "_unix_socket_path", None)
+            if unix_socket_path:
+                self._allowed_hosts_resolved.append(unix_socket_path)
```

Comment:
I don't understand why this change is necessary or what it does. Please explain it.

Reply:
It creates dependancy cycle after cryptography PR.